tox.ini

```diff
@@ -15,6 +15,9 @@ deps =
   pytest-env >= 0.6.2, < 1.0
   requests >= 2.22.0, < 3.0
   requests-mock >= 1.6.0, < 2.0
+  toml >= 0.10
```

Comment:
Please add >= and < bounds as it can help to make the test suite more stable going forward.

Reply:
```suggestion
  toml >= 0.10, < 0.11
```